From 8f52b269ef662822db0d2a7b79e52161403e059a Mon Sep 17 00:00:00 2001
From: Jack Olivieri <jolivieri@anaconda.com>
Date: Wed, 25 Jun 2025 17:18:25 +0200
Subject: [PATCH] fix test

---
 tests/backends/test_mcbackend.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/backends/test_mcbackend.py b/tests/backends/test_mcbackend.py
index e72731af6..7ec8d98a8 100644
--- a/tests/backends/test_mcbackend.py
+++ b/tests/backends/test_mcbackend.py
@@ -27,7 +27,7 @@ try:
 
     from mcbackend.npproto.utils import ndarray_to_numpy
 except ImportError:
-    pytest.skip("Requires McBackend to be installed.")
+    pytest.skip("Requires McBackend to be installed.", allow_module_level=True)
 
 from pymc.backends.mcbackend import (
     ChainRecordAdapter,
-- 
2.39.5 (Apple Git-154)

